src/Config.ts

```diff
@@ -2,6 +2,13 @@ import Level from "./logger/Level";
 import LoggerService from "./logger/LoggerService";
 
 class Config {
+	private dataSetNS: string = "data-c";
```

Comment:
1. Variable not camel cased
2. This variable can't be app global, as it will break third party components that aligned to either their own or the default attributes. namespace

Reply:
Removed from Config.  No longer an issue.